packages/server/graphql/types/AzureDevOpsSearchQuery.ts

```diff
@@ -15,6 +22,10 @@ const AzureDevOpsSearchQuery = new GraphQLObjectType<any, GQLContext>({
       type: new GraphQLNonNull(GraphQLString),
       description: 'The query string, either simple or WIQL depending on the isWIQL flag'
     },
+    projectKeyFilters: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString))),
+      description: 'A list of projects to restrict the search to, if null will search all'
```

Comment:
+1 this is non nullable so we can remove the part of the description that says `if null will search all`